src/pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount.tsx

```diff
@@ -110,30 +100,25 @@ function EnableStep(props) {
                             />
                         )}
                         <MenuItem
-                            title={props.translate('workspace.bankAccount.disconnectBankAccount')}
+                            title={translate('workspace.bankAccount.disconnectBankAccount')}
                             icon={Expensicons.Close}
                             onPress={BankAccounts.requestResetFreePlanBankAccount}
                             wrapperStyle={[styles.cardMenuItem, styles.mv3]}
-                            disabled={Boolean(pendingAction) || !_.isEmpty(errors)}
+                            disabled={Boolean(pendingAction) || !isEmptyObject(errors)}
                         />
                     </OfflineWithFeedback>
                 </Section>
-                {Boolean(props.user.isCheckingDomain) && <Text style={[styles.formError, styles.mh5]}>{props.translate('workspace.card.checkingDomain')}</Text>}
+                {Boolean(user?.isCheckingDomain) && <Text style={[styles.formError, styles.mh5]}>{translate('workspace.card.checkingDomain')}</Text>}
```

Comment:
Remove `Boolean` everywhere , instead use `!!`
```suggestion
                {!!user?.isCheckingDomain && <Text style={[styles.formError, styles.mh5]}>{translate('workspace.card.checkingDomain')}</Text>}
```

Reply:
as commented [here](https://github.com/Expensify/App/pull/34498#discussion_r1455201812)